core/src/test/scala/org/apache/spark/resource/ResourceProfileManagerSuite.scala

```diff
@@ -138,7 +138,7 @@ class ResourceProfileManagerSuite extends SparkFunSuite {
       rpmanager.isSupported(taskProf)
     }.getMessage
     assert(error === "TaskResourceProfiles are only supported for Standalone " +
-      "and Yarn cluster for now when dynamic allocation is disabled.")
+      "and Yarn and Kubernetes cluster for now when dynamic allocation is disabled.")
```

Comment:
nit could change this to "Standalone, YARN, and Kubernetes"

Reply:
Good suggestion, Done.